Annotations/Classes/TextAnnotation/LegibilityTextView.swift

```diff
@@ -0,0 +1,28 @@
+import Cocoa
+
+class LegibilityTextView: NSTextView {
+    
+  override func draw(_ dirtyRect: NSRect) {
+    
+    // this base values are calculated empirically
+    // and their values are used for the proportion to calculate for any font
+    let baseLineWidthSize: CGFloat = 8.0
+    let baseFontSize: CGFloat = 30.0
+          
+    let c = NSGraphicsContext.current?.cgContext
+    
+    let lineWidth: CGFloat
+    if let font = font {
+      lineWidth = font.pointSize * baseLineWidthSize / baseFontSize
+    } else {
+      lineWidth = baseLineWidthSize
+    }
+
+    c!.setLineWidth(lineWidth)
```

Comment:
@ulian-onua maybe let's use guard statement since c is optional.

Reply:
@memstel I agree and will prepare the fix.

Reply:
@mirkokiefer Fixed. Also, changed `c` to `ctx` for better readability